mm2src/mm2_main/src/ordermatch_tests.rs

```diff
@@ -1677,7 +1677,7 @@ fn pubkey_and_secret_for_test(passphrase: &str) -> (String, [u8; 32]) {
     (pubkey, secret)
 }
 
-fn init_p2p_context(ctx: &MmArc) -> (mpsc::Sender<AdexBehaviourCmd>, mpsc::Receiver<AdexBehaviourCmd>) {
+pub(crate) fn init_p2p_context(ctx: &MmArc) -> (mpsc::Sender<AdexBehaviourCmd>, mpsc::Receiver<AdexBehaviourCmd>) {
```

Comment:
This doesn't need crate visibility, why change it?

Reply:
Most likely a leftover.